up-and-running/customization.md

```diff
@@ -1,31 +1,145 @@
 # Customizating Server Configuration
 
 You can customize the server's configuration by visiting npm
-Enterprise's admin console (_http://myreg.mycompany.com:8800_).
+Enterprise's admin console on `:8800`
 
-## Authentication
+  ![Admin Console](/gitbook/images/02-settings.png)
 
-Authentication method.
+## Your First Time Running npm Enterprise
 
-Possible values: [`"GitHub"`](/enterprise/github), [`"Open"`](/enterprise/no-authentication), "LDAP"
+The first time you visit your appliance on port `:8800`, there is some
+initial configuration that you will need to perform:
 
-## Upstream registry
+1. because no certificate has been configured for your appliance, you
+  will need to proceed past a certificate warning in the browser:
 
-Configure npm Enterprise to replicate from an upstream registry.
-This can be useful for creating backups.
+  ![Admin Console](/gitbook/images/01-security-warning.png)
 
-## Storage
+2. next you will be prompted to upload a certificate, or to generate
+  a self-signed certificate (this certificate will be used going forward
+  when accessing the admin console).
+3. on the next screen you will be prompted for the `billing email` and
+  `license key `, which you were provided with when you started your
+  [npm Enterprise trial](https://www.npmjs.com/enterprise).
+4. you will next be prompted to set a password for the npm Enterprise
+  admin console. This password is used by administrators to configure
+  your appliance.
+5. Next, a series of preflight checks will be run. These checks ensure that
+   you are running hardware that meets the requirements.
+6. The docker containers will now begin downloading for npm Enterprise, and
+   you can proceed to [customizing your configuration](/up-and-running/customization.html#customizing-your-configuration).
```

Comment:
The link should probably just be `#customizing-your-configuration` for best support across all formats.